utilities/tools/Set-ReadMeModuleTable.ps1

```diff
@@ -67,14 +67,17 @@ function Set-ReadMeModuleTable {
     # Logic
     $contentArray = Get-Content -Path $FilePath
 
+    # Handle space in the projectname
+    $urlEncodedProjectName = $ProjectName.Replace(' ', '%20')
+
     $tableStringInputObject = @{
         Path           = $ModulesPath
         RepositoryName = $RepositoryName
         Organization   = $Organization
         ColumnsInOrder = $ColumnsInOrder
         SortByColumn   = $SortByColumn
         Environment    = $Environment
-        ProjectName    = $ProjectName
+        ProjectName    = $urlEncodedProjectName
```

Comment:
Alternatively, instead of introducing a new variable, I guess we could also apply the encoding here. i.e.
```suggestion
        ProjectName    = [uri]::EscapeDataString($ProjectName)
```